torchgan/layers/minibatchdiscrimination.py

```diff
@@ -0,0 +1,27 @@
+import torch
+import torch.nn as nn
+
+__all__ = ['MinibatchDiscrimination1D']
+
+# The original paper by Salimans et. al. discusses only 1D minibatch discrimination
+class MinibatchDiscrimination1D(nn.Module):
```

Comment:
Just to conform to the Pytorch Naming Standards
```suggestion
class MinibatchDiscrimination1d(nn.Module):
```